resources/assets/components/pages/AccountPage/Profile/Profile.js

```diff
@@ -10,91 +10,89 @@ import {
   getUtmContext,
   trackAnalyticsEvent,
 } from '../../../../helpers/analytics';
-// import VoterRegStatusBlock from './VoterRegStatusBlock';
+import VoterRegStatusBlock from '../VoterReg/VoterRegStatusBlock';
 
-const Profile = props => (
-  <React.Fragment>
-    <div className="grid-wide">
-      <h1 className="text-xl">Account</h1>
-      <p className="text-gray-600">
-        Update your personal information and manage your account.
-      </p>
-    </div>
-    <div className="grid-wide-2/3 my-6">
-      <h2 className="text-lg">Profile Info</h2>
-      <FormItem
-        title="Name"
-        value={`${props.user.firstName} ${props.user.lastName}`}
-      />
-      {props.user.birthdate ? (
-        <FormItem title="Birthday" value={props.user.birthdate} />
-      ) : null}
-      <FormItem
-        title="Password"
-        value="&#9679; &#9679; &#9679; &#9679; &#9679; &#9679;"
-      />
-      <FormItem title="Email" value={props.user.email} />
-      {props.user.mobile ? (
-        <FormItem title="Phone Number" value={props.user.mobile} />
-      ) : null}
-      <div className="mt-6">
-        <a
-          href={`${env('NORTHSTAR_URL')}/users/${props.user.id}/edit`}
-          target="_blank"
-          rel="noopener noreferrer"
-          className="button"
-        >
-          Edit Profile
-        </a>
+const Profile = props => {
+  return (
```

Comment:
We're not defining any variables here, so this change isn't really needed. It also makes this trickier to review, because so many lines have changed. Could we change this back to returning the component wrapped in parentheses? 
```suggestion
const Profile = props => (
```